src/plugins/data/public/data_sources/datasource_selector/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const SOURCE_PICKER_TITLE = 'available sources';
+export const SOURCE_PICKER_BTN_DEFAULT_TEXT = 'Select a datasource';
+export const SOURCE_PICKER_BTN_DEFAULT_WIDTH = '300px';
+export const SOURCE_PICKER_PANEL_DEFAULT_WIDTH = '500px';
```

Comment:
Why do we need this?

Reply:
we don't need these and actually the whole file. I've already removed all these unused constants.